about.html

```diff
@@ -0,0 +1,172 @@
+        Neque dolores porro quaerat at fugit nesciunt quod, aspernatur,
+        in dignissimos iure dolor expedita assumenda aut numquam tenetur adipisci repellat recusandae.
+      </p>
+      <p class="contact">
+        please contact us per Email for any further question about
+        Drone and Drone taxi Summit 2021
+      </p>
+      <div class="email">
+        <a href="#">Dronesummit2021@outlook.com</a>
+      </div>
+
+    </div>
+  </header>
+
+  <section class="About-logo past-content" id="logo">
+    <h2>Drone and Drones Taxi Global Summit 2021 Logo</h2>
+    <hr class="slim">
+    <img class="event-image"
+      src="https://images.unsplash.com/photo-1591115765373-5207764f72e7?ixid=MnwxMjA3fDB8MHxzZWFyY2h8Nnx8Y29uZmVyZW5jZXxlbnwwfHwwfHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=500&q=60"
+      alt="conference meeting">
+    <p class="description">
+      Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+      Doloremque aliquid culpa esse, repellat a alias sit quas,
+      eaque ab consequatur quidem, praesentium laborum.
+      Laborum rerum quia cupiditate magnam cumque quas!
+    </p>
+    <hr>
+    <p class="logo-detaiil">The logo was designed and decided upon by the organisers of the Summit on may 21 2021</p>
+    <div class="logo">
+      <img src="./asset/image/ddt.png" alt="summit logo" class="summit logo-img">
+    </div>
+    <hr class="line">
+    <div class="contain">
+      <h2>See the past DDT Global Summit </h2>
+      <hr class="slim">
+      <p>take a look at the last two DDT Global Summit which took place in japan and Hong-Kong </p>
+      <div class="activities">
+        <div class="past-first-activity">
+          <div class="first-activity-content first-year flex-column-center">
+            <h3>2020</h3>
+            <p>DDT Global Summit 2020 in Japan</p>
+          </div>
+        </div>
+        <div class="past-second-activity">
+          <div class="second-activity-content flex-column-center">
+            <h3>2019</h3>
+            <p>DDT Global Summit 2020 in Hong-Kong</p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <hr class="full-horizontal-line" />
+  </section>
+
+  <section class="partners" id="sponsor">
+    <h2>Partner</h2>
+    <hr class="slim">
+    <ul>
+      <li><i class="fab fa-firefox fa-3x"></i></li>
+      <li><i class="fab fa-google fa-3x"></i></li>
+      <li><i class="fab fa-apple fa-3x"></i></li>
+      <li><i class="fab fa-bitcoin fa-3x"></i></li>
+      <li><i class="fab fa-airbnb fa-3x"></i></li>
+    </ul>
+  </section>
+  <footer>
+    <div class="copyright">
+      <div class="coperation">
+        <img src="./asset/image/logo_mobile.png" alt="logo-footer">
+      </div>
+      <div class="rights reserve">
+        <p>2021 Drones and Drone Taxi Some Rights Reserved</p>
+        <br>
+        <p class="arabic">انضم إلينا في القمة العالمية للـ دي.دي.تي ونأمل أن نراك هناك</p>
```

Comment:
- [ ] **OPTIONAL**
Even though I understand Arabic very well 😄, it'd be better if you used English for all viewers to understand!

Reply:
yes I understand but the theme I'm trying to bring out is the fact that is going to be held in a country that speaks arabic